src/js/utils/DecorationUtils.js

```diff
@@ -29,12 +15,41 @@ const createNoteWrapper = (
   const sideToRender = cursorAtWidgetAndInsideNote
```

Comment:
I'm a little confused by the math here. 

`priority/maxint` gives me an unpleasant feeling, I wonder if it could be achieved differently (but unlikely I guess)

Reply:
@AWare we could definitely handle priority differently, and better! But I think that change is beyond the scope of this PR.

I think it's fair to say that your confusion is warranted! The intent here is to ensure that decorations with the same `side` property will be rendered in a consistent order – I'll add a comment to make that clear.